app/assess/data.py

```diff
@@ -47,6 +47,21 @@ def get_data(
             )
             return None
 
+def get_assessment_progress(application_metadata):
+    application_ids_list = {
+        'application_ids':
+            [x.get('application_id') for x in application_metadata]
+        }
+    endpoint_url = Config.ASSESSMENT_PROGRESS
+    response = get_data(endpoint=endpoint_url, payload=application_ids_list)
+    if response is not None:
+        [x.update(
+            {"progress":res.get("progress")}
+            ) for res in response for x in application_metadata
+            if res['application_id']==x['application_id']]
```

Comment:
Did you run the pre-commit checks? Indenting looks a bit odd?

Reply:
Haha no I thought I will run at the end as quite few lines looks add on the file so I didn't want to confuse it 😄

Reply:
Done!